contracts/verifiers/ProposalTemplates.sol

```diff
@@ -149,79 +174,71 @@ contract ProposalTemplates is IProposalVerifier, OwnableUpgradeable, Version {
         uint256 start,
         uint256 minEnd,
         uint256 maxEnd
-    ) external view returns (bool) {
+    ) external view {
         if (!exists(pType)) {
             // non-existing template
-            return false;
+            revert VerifierErrors.UnknownTemplate(pType);
         }
         ProposalTemplate memory template = proposalTemplates[pType];
         if (executable != template.executable) {
             // inconsistent executable flag
-            return false;
+            revert VerifierErrors.ExecutableTypeMismatch(executable, template.executable);
         }
         if (minVotes < template.minVotes) {
             // turnout is too small
-            return false;
+            revert VerifierErrors.MinVotesTooSmall(minVotes, template.minVotes);
         }
-        if (minVotes > Decimal.unit()) {
+        uint256 decUnit =  Decimal.unit();
```

Comment:
`Decimal.unit()` is an internal pure library function - there is no need to cache it in a variable